### PR TITLE
f

### PR DESCRIPTION
**Pull request by ancyjoy91**
HI
CAN YOU PLZ CHECK

### CODE REVIEW

File: src/test/java/testClasses/OnlineCalculatorClearButtonTest.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 public class OnlineCalculatorClearButtonTest extends BaseClass {
 
-	//samplete
+	//sampleteh
```
Comment (annjoy91) [inline src/test/java/testClasses/OnlineCalculatorClearButtonTest.java]: CORRECT PLZ

File: src/test/java/testClasses/OnlineCalculatorClearButtonTest.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
 	//test
```
Comment (annjoy91) [inline src/test/java/testClasses/OnlineCalculatorClearButtonTest.java:13]: CHNGE

**Review by annjoy91 — CHANGES_REQUESTED** [review]
PLZ MAKE THESE CHNGES 
OK

File: src/test/java/testClasses/OnlineCalculatorClearButtonTest.java (hunk 2 above)
Comment (ancyjoy91) [reply src/test/java/testClasses/OnlineCalculatorClearButtonTest.java:13]: Comment addressed

**Review by annjoy91 — APPROVED** [review]
PR Approved

**Comment by ancyjoy91** [comment]
merging

**Pull request merged**